### DISCUSSION TITLE
小白问一下如何打开alas就能自动启动调度器

### DISCUSSION THREAD

**Discussion by ttupwz** [LmeSzinc/AzurLaneAutoScript#3501] [Q&A]
是这样的，我平时用nas24小时挂手游的，但是配置不够，不能同时挂着星穹铁道和碧蓝航线，所以用windows的计划命令每天循环启动关闭两款游戏，但是每次启动alas调度器都要手动点一下，想加qq群问下怎么设置启动就运行
![QQ截图20240222000512](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/149664951/4cdb6602-030a-48bd-ab4e-86211fe26b7c)

**Comment by Rin-aki** [comment]
同问，不然只能用Python脚本识别按钮按下了
